qid-batch-runner.yml

```diff
@@ -63,6 +63,32 @@ spec:
               secretKeyRef:
                 name: db-credentials
                 key: password
+          - name: PROJECT_NAME
+            valueFrom:
+              configMapKeyRef:
+                key: project-name
+                name: project-config
+          - name: SFTP_DIRECTORY
+            value: "$(PROJECT_NAME)/upload"
```

Comment:
This might not work well in production. I don't know what the answer is. Maybe we an otherwise version of this .yml with the project name left out?

Reply:
I think we can edit the deployment as required to suit a better SFTP_DIRECTORY. Should be fine.